Sources/SwiftDiagnostics/FixIt.swift

```diff
@@ -46,3 +46,48 @@ public struct FixIt {
     self.changes = changes
   }
 }
+
+public extension FixIt {
+  var edits: [SourceEdit] {
+    var existingEdits = [SourceEdit]()
+    for change in changes {
+      let edit = edit(from: change)
+      var isOverlapping = false
+      for existingEdit in existingEdits {
+        // The edit overlaps with the previous edit. We can't apply both
+        // without conflicts. Apply the one that's listed first and drop the
+        // later edit.
+        if edit.range.overlaps(existingEdit.range) {
+          isOverlapping = true
+          break
+        }
+      }
+      if !isOverlapping {
+        existingEdits.append(edit)
+      }
+    }
+    return existingEdits
+  }
+
+  private func edit(from change: Change) -> SourceEdit {
```

Comment:
You could make this a `private` extension on `SourceEdit`. I think it reads nicer that way.

Reply:
Sorry, I meant to make it `private` extension on `FixIt.Change` so you can write `change.edit` instead of `SourceEdit.edit(from: change)`. Or, alternatively, we could have an initializer on `SourceEdit` that takes a `FixIt.Change`. I don’t have strong opinions so whatever you prefer.